source/dlangbot/github.d

```diff
@@ -27,22 +27,17 @@ string formatComment(in ref PullRequest pr, in IssueRef[] refs, in Issue[] descs
 
     auto app = appender!string();
     app.formattedWrite(
-`Thanks for your pull request, @%s!  We are looking
```

Comment:
Another option is to escape the newline, by putting a `\` at the end of the line.